matchmake-extension/open_participation.go

```diff
@@ -0,0 +1,40 @@
+package matchmake_extension
+
+import (
+	nex "github.com/PretendoNetwork/nex-go"
+	matchmake_extension "github.com/PretendoNetwork/nex-protocols-go/matchmake-extension"
+	common_globals "github.com/PretendoNetwork/nex-protocols-common-go/globals"
+)
+
+func openParticipation(err error, client *nex.Client, callID uint32, gid uint32) {
+	server := commonMatchmakeExtensionProtocol.server
+	session, ok := common_globals.Sessions[gid]
+	rmcResponse := nex.NewRMCResponse(matchmake_extension.ProtocolID, callID)
+	if ok {
```

Comment:
The more idiomatic way of writing these statments in Go is

```suggestion
	if session, ok := common_globals.Sessions[gid]; ok {
```